esp_delta_ota/examples/https_delta_ota/tools/esp_delta_ota_patch_gen.py

```diff
@@ -10,6 +10,8 @@
 import os
 import subprocess
 import re
+import tempfile
```

Comment:
I think these requirements should also be added to https://github.com/espressif/idf-extra-components/blob/master/esp_delta_ota/examples/https_delta_ota/tools/requirements.txt

Reply:
tempfile and hashlib are pre-installed in python, so no need to add these modules in requirements.txt file.